### PR TITLE
modular js

### PR DESCRIPTION
**Pull request by yaredtekle022**
> Created a directory called modules

> Divided my code into modules

> Kept all functionalities of the app without errors

> Displayed the current date below the navigation bar with the luxon library

### CODE REVIEW

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        booklistContainer.addEventListener('click', (e) => {
+          if (e.target.tagName === 'BUTTON') {
+            const id = e.target.getAttribute('id');
+            Book.removeBooks(id);
+            Book.listAllBooks();
+          }
+        });
+      }
```
Comment (thecodechaser) [inline modules/book.js]: - [ ] Please change these functions to arrow functions to follow the ES6 syntax as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @yaredtekle022👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <div class="current-date">
+
+    </div>
+
+    <main class="main"></main>
+
+    <footer>
+      <p>Copyright</p>
+    </footer>
+    <script type="module" src="index.js"></script>
```
Comment (iambenkis) [inline index.html]: - [ ] Good job refactoring your code into ES6 , 👏 . I noticed that your project does not work properly. Books should be rendered to the screen, however there is nothing on the screen 👍 , Kindly fix that to go to the next milestone 👍 

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @yaredtekle022  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (yaredtekle022) [reply index.html]: hello, @iambenkis.
thanks for your comment. The book list is properly rendered on my Chrome and Firefox browsers. Kindly tell me which browser are you using to test my project. i recorded loom video going over my project 

https://www.loom.com/share/069b1a92c1c844739bc97509bb04d3d4

**Review by julie-ify — APPROVED** [review]
Hi @yaredtekle022,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**